clientlib/build.gradle

```diff
@@ -27,15 +27,14 @@ startScripts.enabled = false
 def _artifactId = 'clientlib'
 
 // Dependency versions
-def grpcVersion = '1.30.0' // CURRENT_GRPC_VERSION
```

Comment:
There are 2 dependencies which are still using this variable

Reply:
Good catch, no idea how could I miss it